x-pack/plugins/index_management/public/application/sections/home/data_stream_list/data_stream_list.tsx

```diff
@@ -202,6 +206,35 @@ export const DataStreamList: React.FunctionComponent<RouteComponentProps<MatchPa
               </EuiFlexItem>
             </EuiFlexGroup>
           </EuiFlexItem>
+          <EuiFlexItem grow={false}>
+            <EuiFlexGroup gutterSize="s">
+              <EuiFlexItem grow={false}>
+                <EuiSwitch
+                  label={i18n.translate(
+                    'xpack.idxMgmt.dataStreamListControls.includeManagedSwitchLabel',
+                    {
+                      defaultMessage: 'Include managed',
+                    }
+                  )}
+                  checked={isIncludeManagedChecked}
+                  onChange={(e) => setIsIncludeManagedChecked(e.target.checked)}
+                  data-test-subj="includeManagedSwitch"
+                />
+              </EuiFlexItem>
+
+              <EuiFlexItem grow={false}>
+                <EuiIconTip
+                  content={i18n.translate(
+                    'xpack.idxMgmt.dataStreamListControls.includeManagedSwitchToolTip',
+                    {
+                      defaultMessage: 'Display data streams managed by Ingest Manager',
```

Comment:
```suggestion
                      defaultMessage: 'Display data streams managed by Fleet',
```

Reply:
I think "managed by Fleet" is potentially wrong if this UI will show data streams created by standalone agents (some agents will use local configs and not be managed by Fleet). @ph Can you comment here?

Reply:
Would `Elastic Agent` work in place of `Fleet`? From what I understand, Fleet integrations use the agent but not vice versa. If that's the case, here are a couple suggestions:

* `Display data streams managed by Elastic Agent`
* `Display data streams managed by an agent`

Reply:
Hi @dedemorton and @jrodewig , just some info that could help clarify which data streams we label in this UI: data streams now have `_meta` information and those data streams that were created with an integration in Fleet include both `managed: true` and `managed-by: ingest-manager` in their `_meta` object. Other data streams that are created by agents only have `managed: true` and we are currently not labelling them.

Reply:
Thanks @yuliacech. That helps! I'll update my suggestions accordingly.

Reply:
Thanks for the clarification. In that case, it does make sense to say "managed by Fleet."

Are there plans to change the metadata value from `ingest-manager` to `fleet` now that the UI is named?  Regardless of the underlying setting, you should say Fleet here.